France-ioi/AlgoreaFrontend
# Feature/let user modify his profile

## Description

Fixes #991

## Notes (out of scope, known issues, hints for reviewing code, ...)  (optional)

...

## Test cases

  1. Go to [this page](https://dev.algorea.org/branch/feature/let-user-modify-his-profile)
  3. And I click on Log in / Sign in button
  4. Login with username: update-profile-test, password: q1w2e3r4t5
  5. Then I go to profile page and click on Modify button
  6. Then I see new window with profile form 
  7. Then I change profile info and click on Save button
  8. Then I see modified info in app
